decidim-comments/lib/decidim/comments/api/comment_type.rb

```diff
@@ -27,6 +27,12 @@ module Comments
         }
       end
 
+      field :formattedCreatedAt, !types.String, "The creation date of the comment in relative format" do
+        resolve lambda { |obj, _args, _ctx|
+          obj.friendly_created_at
+        }
+      end
```

Comment:
This is the new parameter I mentioned in the previous comment.

Reply:
it's GraphQL and denormalizing fields doesn't come at any cost, so great for me 👍

Reply:
I think you can simplify this:

```ruby
field :formattedCreatedAt, !types.String, "The creation date of the comment in relative format", property: : friendly_created_at
```